astro/src/content/docs/release-notes/index.mdx

```diff
@@ -45,9 +46,10 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
 
 Looking for release notes older than 1.44.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
-<ReleaseNoteHeading version='1.58.0' releaseDate='July 21, 2025' />
-### Known Issues
-* A bug exists that prevents the Groups page from loading properly. If you use groups, it is recommended that you not upgrade to version `1.58.0` and wait instead for the `1.58.1` patch release.
+<ReleaseNoteHeading version='1.58.2' releaseDate='July 23, 2025' />
+<Aside type="caution">
```

Comment:
Would make more sense to leave in 1.58.0 if possible. (Not sure about 1.58.1, not sure if that version was every public). 

This would be similar to what we did for 1.53.0 and 1.53.1.

Reply:
I think we absolutely should leave existing release notes in place, this number needs to match up to git tags, and GH issue milestones.